packages/core/core/src/services/webhook-store.ts

```diff
@@ -33,31 +34,14 @@ const webhookModel: Model = {
   },
 };
 
-interface DBInput {
-  name: string;
-  url: string;
-  headers: Record<string, string>;
-  events: string[];
-  enabled: boolean;
-}
+type Webhook = Modules.WebhookStore.Webhook;
 
-interface DBOutput {
-  id: string;
-  name: string;
-  url: string;
-  headers: Record<string, string>;
-  events: string[];
+interface DBOutput extends Omit<Webhook, 'id' | 'isEnabled'> {
```

Comment:
That's just a detail but I'm wondering why we're mixing interface (for output) and types (for input).

Is this to allow certain behavior? Otherwise, I would suggest keeping only one syntax (probably type if we don't need the interface features).